src/System.Threading/tests/System.Threading.Tests.builds

```diff
@@ -9,7 +9,7 @@
     </Project>
     <Project Include="System.Threading.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
-      <TestTFMs>net46</TestTFMs>
+      <TestTFMs>net463</TestTFMs>
```

Comment:
I'm a bit confused. I thought net46 mapped to desktop 4.6 surface and net463 mapped to desktop 4.6.3 surface. It would be useful to run tests for newly exposed APIs against the desktop framework as well, to make sure any differences in behavior are understood. If net46 maps to the ns1.3 surface, I don't see how that would be possible.